### PR TITLE
Adding Tracking Catalog doc

### PR DESCRIPTION
**Pull request by elyerry**

### CODE REVIEW

**Review by joe-miskolczy — APPROVED** [review]
Seems good to me

File: docs/pipeline-components-and-applications/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Pipeline Components and Applications"
 date: "2020-03-24"
-sidebar_position: 130
+sidebar_position: 140
```
Comment (paulboocock) [inline docs/pipeline-components-and-applications/index.md:4]: I think I noticed @stanch already mentioned this to you but you can simply insert your new pages in between the existing ones.

**Review by paulboocock — APPROVED** [review]
Looking good! I made a couple of tiny comments, not essential to fix them.

File: docs/data-discovery/tracking-catalog/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+![](images/TC-Landing-page.png)
+
+
+If you select a particular Event, you can see the details of the Events, what properties are tracked for this event, described in plain English therefore easily consumable by a non-technical audience. You can also see any validation criteria associated with this event. The catalog will also display all canonical properties which are also tracked out of the box, when you use Snowplow trackers.
+
+![](images/event-details.png)
+
+
+![](images/canonical-properties.png)
```
Comment (paulboocock) [inline docs/data-discovery/tracking-catalog/index.md:41]: Just noticed one other little thing, the back to back images are a little tricky to look at when looking at the doc page, I'd perhaps suggest moving the sentence from the above paragraph to be in between these images:

> The catalog will also display all canonical properties which are also tracked out of the box, when you use Snowplow trackers.

File: docs/data-discovery/tracking-catalog/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+* * *
+
+
+## Discovery through the Tracking Catalog
+
+
+Often, data consumers only have high level information to guide their data discovery process, and are not aware of the exact name of the Data Structure used for tracking certain data.
+This is why the Tracking Catalog search function will map results of a search to various potential properties. 
+
+When a search is carried out, the Tracking Catalog will try and find results matching the searched text in both the Data Structure name, its description, its properties's names and potential enumeration of values.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:68]:
```suggestion
When you search through the Tracking Catalog, it will look for matches in the Data Structure names, descriptions and fields names. In addition, for fields with a fixed set of allowed values (e.g. a `stage` field with values `checkout` and `payment`) Tracking Catalog will look for matches in the values.
```

Comment (stanch) [reply docs/data-discovery/tracking-catalog/index.md:68]: Editing for clarity

File: docs/data-discovery/tracking-catalog/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+If a match is found, it will highlight the reason behind the match.
+
+
+![](images/search.png)
+
+
+The Tracking Catalog will provide your team with easy and self serve access to this data, creating greater transparency and autonomy.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:75]: I think placed here at the bottom of the page this paragraph loses its punch. I would either put it at the top, in the “Introduction” section, or remove entirely (on the assumption that the benefits are self-evident to anyone who’s read the page).

File: docs/data-discovery/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Data Discovery"
```
Comment (stanch) [inline docs/data-discovery/index.md:2]: This header is unlike the others in two ways:
* It’s not using the gerund (“*Managing* data quality,” “*Modeling* your data”).
* It has the second word capitalized.

At the least, we should change to `Data discovery`. But if you are ok with `Discovering data`, `Exploring your data`, `Exploring your tracking` or something like that, it would bring even more consistency.

File: docs/data-discovery/tracking-catalog/index.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+Discovery and understanding of the data your organisation is tracking through Snowplow is essential to enable teams to unlock real value from that data.
+We know that in many cases, Data Analysts, Data Scientists and Product Managers struggle to find out what is being tracked, and what the data means. They normally have to rely on tribal knowledge and source this information directly with the front end developers who instrumented the tracking, or raise a request to the Data Team.
+
+However, through the Tracking Catalog, all users can have a holistic view of all their tracking in a always up-to-date and easy to consume view.
+
+
+* * *
+
+
+## Browsing the Tracking Catalog
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:20]: Can we make this more goal-oriented? Like “Understanding what data is tracked”? That would help a lot with search and SEO

File: docs/data-discovery/tracking-catalog/index.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+
+**_NOTE:_** Please note that the Tracking Catalog is available only for BDP Customers on selected Tiers. If you're unsure about your eligibility, please get in touch with your Customer Success Manager.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:23]: This note should be the first thing on the page.

File: docs/data-discovery/tracking-catalog/index.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+Once logged in Snowplow Console, you will see a new link on the left hand side navigation, called Tracking Catalog.
```
[the remark this answers is not in the document]
Comment (stanch) [reply docs/data-discovery/tracking-catalog/index.md:25]: I.e. remove “new”

File: docs/data-discovery/tracking-catalog/index.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+If you select a particular Event, you can see the details of the Events, what properties are tracked for this event, described in plain English therefore easily consumable by a non-technical audience. You can also see any validation criteria associated with this event. The catalog will also display all canonical properties which are also tracked out of the box, when you use Snowplow trackers.
+
+![](images/event-details.png)
+
+
+![](images/canonical-properties.png)
+
+
+If you scroll down the page, you will see an Event map. 
+This map display what entities are being tracked with this event. This information is really valuable to understand the connections between the data.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:45]:
```suggestion
This map displays what entities are being tracked with the event. This information is really valuable to understand the connections between the data.
```

File: docs/data-discovery/tracking-catalog/index.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+It also helps data consumers to make sense of how data is structured in the data warehouse and make it easier for them to understand how to query it.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:46]:
```suggestion
It also helps data consumers to make sense of how data is structured in the data warehouse and makes it easier for them to understand how to query it.
```
Does it?

File: docs/data-discovery/tracking-catalog/index.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+
+![](images/event-map.png)
+
+
+Similarly, you can navigate to an Entity, by simply selecting one from the map or the list view, and explore details of it structure and what Events it is being tracked with.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:52]:
```suggestion
Similarly, you can navigate to an Entity by selecting one from the map or the list view. This shows you how it’s structured and what Events include it.
```

File: docs/data-discovery/tracking-catalog/index.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+---
+
+
+## Introducing the Tracking Catalog
+
+
+Discovery and understanding of the data your organisation is tracking through Snowplow is essential to enable teams to unlock real value from that data.
+We know that in many cases, Data Analysts, Data Scientists and Product Managers struggle to find out what is being tracked, and what the data means. They normally have to rely on tribal knowledge and source this information directly with the front end developers who instrumented the tracking, or raise a request to the Data Team.
+
+However, through the Tracking Catalog, all users can have a holistic view of all their tracking in a always up-to-date and easy to consume view.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:14]:
```suggestion
However, through the Tracking Catalog, all users can have a holistic view of all their tracking in an always up-to-date and easy to consume view.
```

File: docs/data-discovery/tracking-catalog/index.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+![](images/Tracking-Catalog-Nav.png)
+
+
+Once you navigate to it, you will be presented with a view showing all the Events and Entities that have been tracked your Snowplow production pipelines in the past 90 days.
+
+![](images/TC-Landing-page.png)
+
+
+If you select a particular Event, you can see the details of the Events, what properties are tracked for this event, described in plain English therefore easily consumable by a non-technical audience. You can also see any validation criteria associated with this event. The catalog will also display all canonical properties which are also tracked out of the box, when you use Snowplow trackers.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:36]: > If you select a particular Event, you can see the details of the Events

I am not sure I understand what this means. Should it be “Event” (singular) the second time as well?

File: docs/data-discovery/tracking-catalog/index.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+
+
+## Discovery through the Tracking Catalog
+
+
+Often, data consumers only have high level information to guide their data discovery process, and are not aware of the exact name of the Data Structure used for tracking certain data.
+This is why the Tracking Catalog search function will map results of a search to various potential properties. 
+
+When a search is carried out, the Tracking Catalog will try and find results matching the searched text in both the Data Structure name, its description, its properties's names and potential enumeration of values.
+If a match is found, it will highlight the reason behind the match.
```
Comment (stanch) [inline docs/data-discovery/tracking-catalog/index.md:69]:
```suggestion
You can find the reason behind the match on the right.
```

**Review by stanch — CHANGES_REQUESTED** [review]

File: docs/data-discovery/tracking-catalog/index.md (hunk 12 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:14]: done

File: docs/data-discovery/tracking-catalog/index.md [hunk 15]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Data Discovery through the Tracking Catalog"
+date: "2022-10-26"
+sidebar_position: 0
+---
+
+
+## Introducing the Tracking Catalog
+
+
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:10]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:25]: done

File: docs/data-discovery/tracking-catalog/index.md [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+## Browsing the Tracking Catalog
+
+
+**_NOTE:_** Please note that the Tracking Catalog is available only for BDP Customers on selected Tiers. If you're unsure about your eligibility, please get in touch with your Customer Success Manager.
+
+Once logged in Snowplow Console, you will see a new link on the left hand side navigation, called Tracking Catalog. 
+
+
+![](images/Tracking-Catalog-Nav.png)
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:28]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 2 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:41]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 9 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:45]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 11 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:52]: modified

File: docs/data-discovery/tracking-catalog/index.md [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+This map display what entities are being tracked with this event. This information is really valuable to understand the connections between the data.
+It also helps data consumers to make sense of how data is structured in the data warehouse and make it easier for them to understand how to query it.
+
+
+![](images/event-map.png)
+
+
+Similarly, you can navigate to an Entity, by simply selecting one from the map or the list view, and explore details of it structure and what Events it is being tracked with.
+
+For both Events and Entity, the Tracking Catalog also displays a log of version changes to help users understand how the data is evolving through time.
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:54]: done

File: docs/pipeline-components-and-applications/index.md (hunk 1 above)
Comment (joe-miskolczy) [reply docs/pipeline-components-and-applications/index.md]: done

File: docs/data-discovery/tracking-catalog/index.md [hunk 18]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Data Discovery through the Tracking Catalog"
+date: "2022-10-26"
+sidebar_position: 0
+---
+
+
+## Introducing the Tracking Catalog
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: implemented but to be discussed

File: docs/data-discovery/tracking-catalog/index.md [hunk 19]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Data Discovery through the Tracking Catalog"
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: implemented but to be discussed

File: docs/data-discovery/tracking-catalog/index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: - Capitalization ✅
- Links ✅
- Synchronization of "fields" and "properties" ❌

File: docs/data-discovery/tracking-catalog/index.md (hunk 10 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: Implemented, but to be answered

File: docs/data-discovery/tracking-catalog/index.md (hunk 7 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: We would prefer to leave "properties" like it is, as the feature in Console and the marketing material uses this terminology and later on align the docs on a unified name.

File: docs/data-discovery/tracking-catalog/index.md [hunk 20]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Similarly, you can navigate to an Entity, by simply selecting one from the map or the list view, and explore details of it structure and what Events it is being tracked with.
+
+For both Events and Entity, the Tracking Catalog also displays a log of version changes to help users understand how the data is evolving through time.
+
+![](images/versions.png)
+
+
+* * *
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:52]: We would prefer to keep them in.

File: docs/data-discovery/tracking-catalog/index.md [hunk 21]
```diff
@@ -0,0 +1,79 @@
...
+
+
+## Discovery through the Tracking Catalog
+
+
+Often, data consumers only have high level information to guide their data discovery process, and are not aware of the exact name of the Data Structure used for tracking certain data.
+This is why the Tracking Catalog search function will map results of a search to various potential properties.
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:57]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 3 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:59]: implemented with field -> property replacement

File: docs/data-discovery/index.md (hunk 5 above)
Comment (joe-miskolczy) [reply docs/data-discovery/index.md]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 6 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:22]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 10 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: In our opinion it does.

File: docs/data-discovery/tracking-catalog/index.md (hunk 11 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 4 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:64]: We would still like to keep it here for now and maybe do a second round later.

File: docs/data-discovery/tracking-catalog/index.md [hunk 22]
```diff
@@ -0,0 +1,79 @@
...
+![](images/versions.png)
+
+
+* * *
+
+
+## Discovery through the Tracking Catalog
+
+
+Often, data consumers only have high level information to guide their data discovery process, and are not aware of the exact name of the Data Structure used for tracking certain data.
```
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:56]: Please can you let it slide for now, we would rather give the whole thing an overhaul with this mindset than updating a single paragraph but this feature is being rolled out at the moment.

File: docs/data-discovery/tracking-catalog/index.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 14 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md:60]: done

File: docs/data-discovery/tracking-catalog/index.md (hunk 13 above)
Comment (joe-miskolczy) [reply docs/data-discovery/tracking-catalog/index.md]: done

**Review by stanch — APPROVED** [review]
